platformio.ini

```diff
@@ -769,9 +769,10 @@ build_flags       = ${common.build_flags}
   -DHAVE_HWSERIAL3
   -DPIN_SERIAL2_RX=PD_6
   -DPIN_SERIAL2_TX=PD_5
+  -IMarlin/src/HAL/STM32
```

Comment:
It looks like this was added so a pointy include will find `SoftwareSerial.h`. Instead of this, we should use a quotey include, making it clear that the local copy is intended, and no library copy is expected.

Unless! A library copy is sometimes expected.

Reply:
That has to be there so that TMCStepper can find it.

Reply:
Yep, due to the use of the pointy brace style include. I see why these environments are taking that approach. A comment should mention that this is for the benefit of `SoftwareSerial.h` specifically.